src/plugins/FingerPrint.js

```diff
@@ -106,9 +110,9 @@ module.exports = class FingerPrint extends OverlayPlugin {
                                 }
                             }
                             if (value) {
-                                this.toolbarBtn.addClass('fingerprint-autoValidation');
+                                this.toolbarBtn.setIndicator('actif');
```

Comment:
```suggestion
                                this.toolbarBtn.setIndicator('active');
```
Same here, so I guess not exactly a typo but it should be fixed none the less